tests/integration/popup/form_submission_core.js

```diff
@@ -0,0 +1,32 @@
+asyncTest( "Form submission from popup works", function() {
+	var eventNs = ".formSubmissionFromPopupWorks";
+
+	expect( 4 );
```

Comment:
Move this to be first the expect should be first thing in the test ( i know we are not consistent about this now but no time like the present to start and this is what UI does )